### PR TITLE
All: adds a new const Symbol for the admin key

### PR DESCRIPTION
**Pull request by gangov**
adds a new function that will copy the admin address from `DataKey::Admin` to the new `const ADMIN`

### CODE REVIEW

**Comment by gangov** [comment]
>we have a discrepancy in the storage types used to keep the admin value, I think we should settle on just one type.

namely - all the contracts are using `persistent` storage for `DataKey::Admin`, while `multihop` and `pool_stable` are using `instance`

**Comment by ueco-jb** [comment]
> namely - all the contracts are using persistent storage for DataKey::Admin, while multihop and pool_stable are using instance

Actually now I think this should be stored in an instance type of storage. This is a small string and it's necessary for the contract to run.

File: contracts/factory/src/contract.rs [hunk 1]
```diff
@@ -489,6 +491,13 @@ impl FactoryTrait for Factory {
             stake_portfolio,
         }
     }
+
+    fn migrate_admin_key(env: Env) -> Result<(), ContractError> {
+        let admin = get_config(&env).admin;
+        env.storage().persistent().set(&ADMIN, &admin);
```
Comment (ueco-jb) [inline contracts/factory/src/contract.rs:497]:
```suggestion
        env.storage().instance().set(&ADMIN, &admin);
```

**Review by ueco-jb — COMMENTED** [review]
Please also add new function `get_admin`/`set_admin` which loads that data.
Then rename the existing ones (that are using `DataKey::Admin`) to the `_old` meaning:
- the current `get_admin` becomes `save_`/`get_admin_old`
- you replace usage everywhere with `save_`/`get_admin_old`
- you create a new function `save_`/`get_admin` which uses the `ADMIN` symbol
- set a new version in the contracts so we would now it's a major change

**Review by ueco-jb — APPROVED** [review]
LGTM

**Pull request merged**